.github/workflows/_generate-changelog.yml

```diff
@@ -144,7 +144,7 @@ jobs:
                     changie batch ${{ needs.version.outputs.base }}
                 fi
                 changie merge
-                git status
```

Comment:
I verified this is the behavior of `--all-files` by:
- create a changelog entry with `changie`
- add extra newlines at the end of the entry
- do NOT run `git add .`
- run `pre-commit run --all-files`
- notice this passes
- DO run `git add.`
- run `pre-commit run --all-files`
- the file triggers the newline check, fixing the newlines